### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by GarySkt**
projects create

### CODE REVIEW

File: app/piscis_app_client/.gitignore [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+*.pyc
+*.swp
+.DS_Store
+.atom/
+.buildlog/
+.history
+.svn/
+
+# IntelliJ related
+*.iml
```
Comment (GarySkt) [inline app/piscis_app_client/.gitignore:13]: oe asdasd

File: app/piscis_app_client/.gitignore [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+# Flutter/Dart/Pub related
+**/doc/api/
+**/ios/Flutter/.last_build_id
+.dart_tool/
+.flutter-plugins
+.flutter-plugins-dependencies
+.packages
+.pub-cache/
+.pub/
```
Comment (LuisEdMB) [inline app/piscis_app_client/.gitignore:31]: kljljl

Comment (GarySkt) [reply app/piscis_app_client/.gitignore:31]: aea p hazlo tu

File: app/piscis_app_client/.gitignore (hunk 1 above)
Comment (LuisEdMB) [reply app/piscis_app_client/.gitignore:13]: yo he colocado eso para tantoad

Comment (GarySkt) [reply app/piscis_app_client/.gitignore:13]: aea manito

**Review by LuisEdMB — DISMISSED** [review]

**Review by LuisEdMB — DISMISSED** [review]

File: app/piscis_app_client/.gitignore [hunk 3]
```diff
@@ -0,0 +1,46 @@
+# Miscellaneous
+*.class
+*.log
```
Comment (GarySkt) [inline app/piscis_app_client/.gitignore:3]: oe

**Review by LuisEdMB — DISMISSED** [review]

**Review by LuisEdMB — APPROVED** [review]

**Pull request merged**